Dockerfile.olmo

```diff
@@ -0,0 +1,115 @@
+ARG CUDA
```

Comment:
do we want to add documentation about building with this image or nah? We had it before.